content/structure/TextEntity.md

```diff
@@ -0,0 +1,60 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/TextEntity
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square)
+
+```c#
+public struct TextEntity
+```
+
+`TextEntity` stores information about special entities (like mentions or hashtags) that can be found in a text.
+
+> This is only relevant for the Twitter API, as Mastodon provides an HTML-string where we can probably just replace the entity-links with our values.
```

Comment:
Replacing values may not be entirely trivial. We might be able to intercept the URL handling and just parse the platform's URLs rather than using custom URLs (as we do now) but we'd have to be careful not to intercept the "open in browser" link as well.